creator-node/.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
just curious, is this config something you just picked, or did one of the breakout jam sessions come up with this
good w me either way 👌

Reply:
this is straight from the client repo https://github.com/AudiusProject/audius-client/blob/master/.prettierrc.js